itn-eval-metrics.Rmd

```diff
@@ -535,10 +536,34 @@ ggplot2::ggsave(file.path("plots", "itn_software_website_traffic.png"), width =
 ```{r}
  #download_stats <- cran_stats(c("ottrpal", "conrad", "ari", "text2speech"))
 
- #download_stats %>% dplyr::summarize(download_total = sum(downloads))
+download_stats <- cranlogs::cran_downloads(packages = c("ottrpal", "conrad", "ari", "text2speech"), from = "2017-08-31", to = "last-day") #first version of ari published on 2017-08-31
 
- #download_stats %>% dplyr::group_by(package) %>%
- #  dplyr::summarize(download_total = sum(downloads))
+download_stats %>% dplyr::summarize(download_total = sum(count))
+
+download_stats %>% dplyr::group_by(package) %>%
+   dplyr::summarize(download_total = sum(count))
+
+download_stats_to_plot <- download_stats %>% separate(date, into=c("year", "month name", "day"), sep = "-") %>% unite("Month", c("year", "month name"), sep='-', remove=TRUE) %>%  #cran_downloads returns the daily downloads, and so doing some prep work to turn that into monthly downloads
```

Comment:
This reminds me I should run this whole thing through `styler::style_file()` But I can do that on another branch.